jwst/datamodels/schemas/darkMIRI.schema.yaml

```diff
@@ -0,0 +1,27 @@
+title: MIRI Dark current data model
+allOf:
+- $ref: core.schema.yaml
+- $ref: referencefile.schema.yaml
+- type: object
+  properties:
+    data:
+      title: MIRI Dark current array
+      fits_hdu: SCI
+      default: 0.0
+      ndim: 4
+      datatype: float32
+    dq:
+      title: 4-D data quality array for all planes
+      fits_hdu: DQ
+      default: 0
+      ndim: 4
+      datatype: uint32
```

Comment:
Does this need to be as big as int32? If only a few DQ bits are used in the reference data, this could be reduced to something like uint8. That still gives you 8 separate DQ bits to work with and saves a lot of space.